Makefile

```diff
@@ -1,38 +1,49 @@
 OS := BigOS
 
-CC := riscv64-unknown-elf-gcc
-AC := riscv64-unknown-elf-ac
-LD := riscv64-unknown-elf-ld
-VM := qemu-system-riscv64
-OBJDUMP := riscv64-unknown-elf-objdump
-OBJCOPY := riscv64-unknown-elf-objcopy
-
-CFLAGS := -Wall -Wextra -ffreestanding -nostdlib -g -O0
-LDFLAGS := -T linker.ld -nostdlib
-VMFLAGS := 
+export CC := riscv64-elf-gcc
+export AS := riscv64-elf-as
+export LD := riscv64-elf-ld
+export VM := qemu-system-riscv64
+export OBJDUMP := riscv64-elf-objdump
+export OBJCOPY := riscv64-elf-objcopy
```

Comment:
Unused in this makefile, could be extracted into `common.mk` to be later `included` where needed.
The same goes for `_FLAGS`.